### ISSUE TITLE
Make A11y metadata available before download

### ISSUE THREAD

**Issue by StephDriver** [openlibhums/janeway#4510]
As part of the upcoming EU A11y act, information on how accessible an ebook is should be made available to users prior to download of that publication.

It is unclear as to what extent this applies to articles, books, prepubs etc within Janeway, however as our approach is that open access = access for all, we take standards as the base, not the target.  Therefore we should look at including information about accessibility prior to downloading.

This could be through [ONIX Code List 196.](https://kb.daisy.org/publishing/docs/metadata/onix.html#:~:text=Explanation,accessible%20features%20the%20EPUB%20offers.)

**Comment by ajrbyers** [comment]
ONIX is a standard that applies only to books so wont be appropriate for articles.

**Comment by StephDriver** [comment]
I've heard there is debate on what 'ebook' means for the EU Act, with individual articles likely exempt, but not necessarily anything that is more "booklike" - e.g. a collections of articles.  This is something we may need to clarify... or at least keep an eye as to how it develops.

But regardless of the legal part, I very much support the general principle of accessibility information about a downloadable file, of any kind, being available to a reader before they download it - as this could be relevant to their decision on whether to download that file or not. 

> ONIX is a standard that applies only to books so wont be appropriate for articles.
I thought it was useful for epubs regardless of length (ie. article v book), but I may have misread that bit then.
